Update project metadata (readme, license, setup.py, etc) (#11)
ESMValGroup/ESMValTool_sample_data: CONTRIBUTING.md

# Contributing guidelines

We welcome any kind of contribution to our software, from simple comment or question to a full fledged [pull request][1].

A contribution can be one of the following cases:

1. you have a question;
2. you think you may have found a bug (including unexpected behavior);
3. you want to make some kind of change to the code base (e.g. to fix a bug, to add a new feature, to update documentation).

The sections below outline the steps in each case.

## You have a question

1. use the search functionality [here][2] to see if someone already filed the same issue;
2. if your issue search did not yield any relevant results, make a new issue;
3. apply the "Question" label; apply other labels when relevant.

## You think you may have found a bug

1. use the search functionality [here][2] to see if someone already filed the same issue;
2. if your issue search did not yield any relevant results, make a new issue, making sure to provide enough information to the rest of the community to understand the cause and context of the problem. Depending on the issue, you may want to include:

>     - the [SHA hashcode][3] of the commit that is causing your problem;

Reviewer: ```suggestion
```
this might be a bit too technical for most people who use this package

    - some identifying information (name and version number) for dependencies you're using;
    - information about the operating system;
3. apply relevant labels to the newly created issue.

> ## You want to make some kind of change to the code base

Reviewer: ```suggestion
## You want to make a change to the code
```

1. (**important**) announce your plan to the rest of the community _before you start working_. This announcement should be in the form of a (new) issue [here][2];
2. (**important**) wait until some kind of consensus is reached about your idea being a good idea;
3. if needed, create your own feature branch off of the latest master commit. While working on your feature branch, make sure to stay up to date with the master branch by pulling in changes, possibly from the 'upstream' repository (follow the instructions [here][4] and [here][5]);
<!-- 4. make sure the existing tests still work by running ``python setup.py test``; -->
<!-- 5. add your own tests (if necessary); -->
6. update or expand the documentation;

> 7. [push][6] your feature branch to (your fork of) the Python Template repository on GitHub;

Reviewer: ```suggestion
7. [push][6] your feature branch to (your fork of) the ESMValTool_sample_data repository on GitHub;
```
